rvic/parameters.py

```diff
@@ -31,6 +31,16 @@
 # -------------------------------------------------------------------- #
 # Top level driver
 def parameters(config_file, numofproc=1):
+    '''
```

Comment:
I know that functionally it makes no differences, but you use `"""` for the other doc strings. May be nice to standardize (or just tell me "won't fix"